src/beignet/_root.py

```diff
@@ -0,0 +1,144 @@
+from typing import Callable, Optional
+
+import torch
+
+
+def root(
+    f: Callable[[torch.Tensor], torch.Tensor],
```

Comment:
The Beignet rule for imports is non-capitalized items (e.g., functions) are explicit and capitalized items (e.g., constants and classes) are implicit.